<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="10X-D-Hymavathi" name="com.axis.gbm.ppf.testscripts.GBM_TS2_PPFInerSolTransfer" tests="1" failures="0" timestamp="25 Sep 2014 14:34:11 GMT" time="175.075" errors="1">
  <testcase name="PPFInterSolTransferTransaction" time="175.075" classname="com.axis.gbm.ppf.testscripts.GBM_TS2_PPFInerSolTransfer">
    <error type="java.lang.AssertionError" message="Unable to login. Error Message:Invalid user details">
      <![CDATA[java.lang.AssertionError: Unable to login. Error Message:Invalid user details
	at org.testng.Assert.fail(Assert.java:94)
	at com.axis.gbm.ppf.testscripts.GBM_TS2_PPFInerSolTransfer.PPFInterSolTransferTransaction(GBM_TS2_PPFInerSolTransfer.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- PPFInterSolTransferTransaction -->
</testsuite> <!-- com.axis.gbm.ppf.testscripts.GBM_TS2_PPFInerSolTransfer -->
